L/libunicorn/build_tarballs.jl

```diff
@@ -0,0 +1,36 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libunicorn"
+version = v"1.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/oblivia-simplex/unicorn.git", "cb2c6c18e713806aeedbc4a71e18f541b77cd903")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/unicorn
+make -j${nproc}
+make PREFIX=${prefix} install
+install_license COPYING*
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = filter(Sys.islinux, supported_platforms())
```

Comment:
I wonder why you restrict this to Linux -- the README of Unicorn explicitly lists "Native support for Windows & *nix (with Mac OSX, Linux, *BSD & Solaris confirmed)". Are there build issues on these?

Reply:
I'm having trouble getting it to build for windows, whether I use my fork of unicorn or the upstream version. Everything else is building fine, now, though, and I've updated the build script accordingly.